util/etcd/election_test.go

```diff
@@ -65,18 +64,16 @@ func everyoneAgreesOnMaster(ctx context.Context, t *testing.T, want string, elec
 	t.Helper()
 	for _, e := range elections {
 		for {
-			log.Print("Getting master")
-			m, err := e.GetCurrentMaster(ctx)
+			got, err := e.GetCurrentMaster(ctx)
 			if err == concurrency.ErrElectionNoLeader {
```

Comment:
Does this test need updating for `util.ErrNoLeader`

Reply:
thanks!